File: Práctica 1/S1/src/s1/factoriaMontana.java
package s1;

import java.util.ArrayList;

public class factoriaMontana extends factoriaCarrerayBicicleta {

	public carreraMontana crearCarrera(int codigo) {
            carreraMontana carrera = new carreraMontana(codigo);
            return carrera;
	}

	public bicicletaMontana crearBicicleta(int id) {
            bicicletaMontana bicicleta = new bicicletaMontana(id);
            return bicicleta;
	}
}

File: Práctica 1/S1R/lib/FactoriaMontana.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.
module Ruby
  class FactoriaMontana < FactoriaCarrerayBicicleta
    def crearCarrera()
      carrera = CarreraMontana.new.clonar
      return carrera
    end
    def crearBicicleta()
      bicicleta = BicicletaMontana.new.clonar
      return bicicleta
    end
  end
end
